Server/JokesOnYou.Web.Api/Models/Request/UserRegisterDTO.cs

```diff
@@ -8,9 +9,13 @@ public class UserRegisterDto
         [EmailAddress]
         public string Email { get; set; }
         [Required]
+        [MinLength(4)]
         public string UserName { get; set; }
         [Required]
         public string Password { get; set; }
-
+        [Required]
```

Comment:
Doesnt have to be required, can be false by default

Reply:
We probably should, i dont think its legal showing nsfw stuff to people under 18 or something

Reply:
nsfw does not mean Sexy stuff per se and we wont even allow like naked stuff. (not that we have pictures yet)
Should just be there for like Swear words and stuff?

Reply:
Done